### PR TITLE
Fix #33630. Fix cannot set 0 as value on files_external through OCC command

### LINKED ISSUE #33630: Cannot set  `filesystem_check_changes` to never through occ command
This is a bug in the `files_external` app in which if we want to set `filesystem_check_changes` to never, it doesnot set any value. Instead it just returns whatever the current value is. 

## How To Reproduce:
1. Connect to an external storage with owncloud
2. Run the following:
```
sudo -u www-data ./occ files_external:option <mount_id> filesystem_check_changes 0
```

## Expected Outcome:
1. No output is printed through occ command.
2. `Check for changes` is set to `Never`.
(On Settings->Storage->External Storage->GearIcon :gear: ) on the webUI

## Actual Outcome:
1. Prints already set config.
2. Doesnot change anything on the webUI

## Bug Analysis
This seems to be due to [this conditional check](https://github.com/owncloud/core/blob/8cd722356cf81df22699a1e17450f2b034235b23/apps/files_external/lib/Command/Config.php#L75-L79).

### PR DESCRIPTION
**Pull request by skshetry**
## Description

The OCC command didn't allow us to set value 0 due to the use of `if ($value)` checks. Now, This PR modifies it to check for null value only.

## Related Issue

- Fixes #33630 

## How Has This Been Tested?

- Locally

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Comment by phil-davis** [comment]
@skshetry are there acceptance tests for this yet?

**Comment by skshetry** [comment]
@phil-davis No, I don't think we have tested occ commands for any apps that are in core. Here's the broader issue: #33051. 

This doesn't seem to have any unit tests also. So, I'm thinking of covering the changed code, and needs to figure that out. I'll write acceptance tests for this also.

File: apps/files_external/lib/Command/Config.php [hunk 1]
```diff
@@ -72,7 +72,8 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		}
 
 		$value = $input->getArgument('value');
-		if ($value) {
+
+		if ($value !== null) {
```
Comment (PVince81) [inline apps/files_external/lib/Command/Config.php]: classic PHP :man_facepalming:

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
@skshetry nice find. Please backport this bugfix to stable10

**Comment by skshetry** [comment]
Backport for `stable10` on #33643

**Pull request merged**